docker-compose.yml

```diff
@@ -4,36 +4,78 @@ services:
     image: postgres:9.3
     container_name: api-java-postgres
 
-  possum:
-    image: registry.tld/possum:0.1.0-stable
-    container_name: api-java-possum
+  conjur:
+    image: cyberark/conjur:0.1.0-stable
+    container_name: api-java-conjur
     command: server -a cucumber
     environment:
+      PORT: 3000
       DATABASE_URL: postgres://postgres@postgres/postgres
       CONJUR_DATA_KEY: "W0BuL8iTr/7QvtjIluJbrb5LDAnmXzmcpxkqihO3dXA="
+    ports:
+      - "3000:3000"
+    expose:
+      - "3000"
     depends_on:
       - postgres
 
+  conjur-proxy-nginx:
+    build:
+      context: .
+      dockerfile: Dockerfile.nginx
+    entrypoint: nginx-debug -g 'daemon off;'
+    container_name: api-java-proxy
+    environment:
+        TERM: xterm
+    ports:
+    - "8080:80"
+    - "8443:443"
+    expose:
```

Comment:
`expose` should be done in the `Dockerfile` as `EXPOSE`, not here. We should only need `ports`.

Reply:
Good point, I thought I need the expose in order to access from outside the docker-compose env, but the port did the work.
I will delete it